yarn.lock

```diff
@@ -114,6 +114,15 @@
     "@jridgewell/gen-mapping" "^0.3.2"
     jsesc "^2.5.1"
 
+"@babel/generator@^7.19.0":
+  version "7.19.0"
+  resolved "https://registry.yarnpkg.com/@babel/generator/-/generator-7.19.0.tgz#785596c06425e59334df2ccee63ab166b738419a"
```

Comment:
@jibees 
Each update adds another entry for a new version. We have five babel/generator versions in the lockfile here. Is that right? How do we clean this up and keep it clean?

Reply:
It's the nightmare of the dependencies. If you search for `@babel/generator` in `yarn.lock` file, you'll see that some version are needed by some packages, and those packages are needed by some ...

Reply:
That is crazy dependency management. Packages should have a range defined with which they work (like gems) and not lock in a particular version.